src/ucp/rndv/rndv_rkey_ptr.c

```diff
@@ -240,8 +242,9 @@ ucp_proto_rndv_rkey_ptr_mtype_probe(const ucp_proto_init_params_t *init_params)
         return;
     }
 
-    status = ucp_proto_rndv_mtype_init(init_params, &mdesc_md_map,
-                                       &params.super.max_length);
+    /* 2-stage ppln protocols work with host staging buffers only */
+    status = ucp_proto_rndv_mtype_init(init_params, UCS_MEMORY_TYPE_HOST,
```

Comment:
matbe we should check the supported memory types of rkey_ptr lane?

Reply:
why? we can update the code if/when new 2-stage ppln scheme is implemented

Reply:
ok